README.md

```diff
@@ -1,86 +1,118 @@
 # AMD MIGraphX
 
-AMD's graph optimization engine.
+AMD MIGraphX is AMD's graph inference engine to accelerate model inference on AMD GPUs. AMD MIGraphX can be used by
+installing binaries directly or building from source code.
+
+Note that all the following instructions are based on that ROCm has been installed successfully. ROCm installation
+instructions are explained in the [ROCm installation
+guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
+
+## Installing from binaries
+With ROCm installed correctly, MIGraphX binaries can be installed with the following command:
+```
+sudo apt update && sudo apt install -y migraphx
+```
+then the head files and libs are located at `/opt/rocm/include` and `/opt/rocm/lib`, respectively, which can be
+included and linked by adding the corresponding folders to the Makefile.
```

Comment:
This line should be removed. MIGraphX should be linked using the `migraphx::migraphx_c` cmake target. We should add another sections that says:

> To use MIGraphX's C/C++ API in your cmake project, then set `CMAKE_PREFIX_PATH` to your rocm installation and then do(where `myApp` is the cmake target in your project):

```cmake
find_package(migraphx)
target_link_libraries(myApp migraphx::migraphx_c)
```

Reply:
Added a section to mention this part.